netmanager/src/views/pages/Download/Download.js

```diff
@@ -319,7 +328,7 @@ const Download = (props) => {
         dispatch(
           updateMainAlert({
             show: true,
-            message: err,
```

Comment:
This is an object and yet message expects a string. It was throwing a blank screen and some errors. Fixed that.